src/main/scala/io/iohk/ethereum/transactions/PendingTransactionsManager.scala

```diff
@@ -153,13 +159,15 @@ class PendingTransactionsManager(
 
     case RemoveTransactions(signedTransactions) =>
       pendingTransactions.invalidateAll(signedTransactions.map(_.hash).asJava)
+      log.debug("Removing transactions: {}", signedTransactions.map(_.hash.toHex))
```

Comment:
A picky detail.. I see 3 different verbal times: past, imperative and continuous :)